bin/radicale

```diff
@@ -1,31 +0,0 @@
-#!/usr/bin/env python3
```

Comment:
Good change. But I guess, the chance to get this merged is higher in a separate PR.